requirements/doc.txt

```diff
@@ -11,7 +11,7 @@ certifi==2018.11.29       # via requests
 chardet==3.0.4            # via doc8, requests
 doc8==0.8.0
 docutils==0.14            # via doc8, readme-renderer, restructuredtext-lint, sphinx
-edx-sphinx-theme==1.3.0
+edx-sphinx-theme==1.4.0
```

Comment:
@nasthagiri: I tried updating the theme to allow for your custom index page.  Master branch (before my change) is now failing, so this is blocked.  I will probably move this to a separate PR.  It just fixes the sidebar nav.  We can improve this later, or do a more traditional TOC.

Reply:
Moving this to a separate PR.